sigmf-spec.md

```diff
@@ -336,43 +338,58 @@ about the Recording itself.
+| `num_channels`   | false    | uint    | Total number of interleaved channels in the dataset file. If omitted, this defaults to one.|
+| `sha512`         | false    | string  | The SHA512 hash of the Dataset file associated with the SigMF file.|
+| `offset`         | false    | uint    | The index number of the first sample in the Dataset. If not provided, this value defaults to zero. Typically used when a Recording is split over multiple files. All sample indices in SigMF are absolute, and so all other indices referenced in metadata for this recording SHOULD be greater than or equal to this value.|
+| `description`    | false    | string  | A text description of the SigMF Recording.|
+| `author`         | false    | string  | A text identifier for the author potentially including name, handle, email, and/or other ID like Amateur Call Sign. For example "Bruce Wayne <bruce@waynetech.com>" or "Bruce (K3X)".|
+| `meta_doi`       | false    | string  | The registered DOI (ISO 26324) for a Recording's Metadata file.|
+| `data_doi`       | false    | string  | The registered DOI (ISO 26324) for a Recording's Dataset file.|
+| `recorder`       | false    | string  | The name of the software used to make this SigMF Recording.|
+| `license`        | false    | string  | A URL for the license document under which the Recording is offered.|
+| `hw`             | false    | string  | A text description of the hardware used to make the Recording.|
+| `dataset`        | false    | string  | The full filename of the dataset file this Metadata file describes.|
+| `trailing_bytes` | false    | uint    | The number of bytes to ignore at the end of a Non-Conforming Dataset file.|
+| `virtual_dataset`| false    | bool    | Indicates that the dataset is virtual (SigMF Recording is metadata only).|
+| `geolocation`    | false    | GeoJSON `point` object | The location of the Recording system.|
+| `extensions`     | false    | array   | A list of JSON Objects describing extensions used by this Recording.|
+| `collection`     | false    | string  | The base filename of a `collection` with which this Recording is associated.|
 
 ##### The `dataset` Field
+
 The `core:dataset` field in the Global object is used to specify the dataset file that
 this Metadata describes. If provided, this string MUST be the complete filename of the
 dataset file, including the extension. The dataset file must be in the local directory,
 and this string MUST NOT include any aspects of filepath other than the filename.
 
-If this field is omitted, the dataset file MUST be a SigMF Dataset file (NOT a
+If this field is omitted, the dataset MUST have a SigMF compliant Dataset file (NOT a
 Non-Conforming Dataset), and MUST have the same base filename as the Metadata file and
-use the `.sigmf-data` extension.
+use the `.sigmf-data` extension. If it is acceptable to rename the files in the SigMF
+recording, it is RECOMMENDED that SigMF Compliant recordings DO NOT use this field.
```

Comment:
Hm, I don't think I'm following this new sentence in the spec. What are you trying to capture with this bit?

Reply:
basically this field provides an explicit link between metadata and dataset files, in addition to the implicit link that Compliant recordings have. If you define this field for a compliant recording's metadata, and then down the road that recording is renamed, there may now be conflict between the implicit and explicit defined metadata file.